app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -207,10 +203,22 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
               label="Time"
               disabled={!values.publishedOn}
               width="7em"
+              value={!!values.publishedOn ? moment(values.publishedOn).format('HH:mm:ss') : ''}
               placeholder={
                 !!values.publishedOn ? moment(values.publishedOn).format('HH:mm:ss') : 'HH:MM:SS'
               }
-              onChange={(e) => setPublishedOnTime(e.target.value)}
+              onChange={(e) => {
+                const date = new Date(values.publishedOn);
+                const hours = e.target.value?.split(':');
+                if (
+                  !!hours &&
+                  !values.publishedOnTime?.includes('_') &&
+                  values.publishedOnTime !== ''
+                ) {
+                  date.setHours(Number(hours[0]), Number(hours[1]), Number(hours[2]));
+                  setFieldValue('publishedOn', date.toISOString());
```

Comment:
Update the `publishedOn` and the time will be updated by the `React.useEffect(...)` above.